### PR TITLE
Improve readability of README.md

### PR DESCRIPTION
**Pull request by okuta**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -5,6 +5,18 @@
 [![Read the Docs](https://readthedocs.org/projects/cupy/badge/?version=stable)](http://docs.cupy.chainer.org/en/stable/?badge=stable)
 
 # CuPy : NumPy-like API accelerated with CUDA
+[**Website**](http://cupy.chainer.org/)
```
Comment (unnonouno) [inline README.md]: Use https

**Comment by niboshi** [comment]
I think the links to slack teams are not user friendly.
They're actually chainer teams, but users won't notified about that.
Users could make duplicated requests without explicit notice.

**Comment by okuta** [comment]
OK, I remove slack and twitter links

**Comment by niboshi** [comment]
The title "_CuPy : NumPy-like API accelerated with CUDA_" is duplicated.

**Comment by niboshi** [comment]
LGTM!

**Pull request merged**